src/templates/episode.js

```diff
@@ -251,7 +253,12 @@ export default ({ data }) => {
               ))}
             </div>
           </div>
-        )}
+        ) : (
```

Comment:
It's probably obvious that I'm a React n00b, but I couldn't figure out how to extract the <h1> out of the conditional so as not to repeat it - I kept getting a jsx compiler error.

Reply:
Oh ok. I can help you out here. You could do:

```js
          <div>
            <h1>Transcript</h1>
            {transcript ? (
              <h4>
                Help improve this transcript on{' '}
                <a href={transcriptLink}>GitHub</a>
              </h4>
            ) : (
              <h4>
                Help by adding a transcript on{' '}
                <a href={transcriptsLink}>GitHub</a>
              </h4>
            )}
```

Reply:
oh ha - that has nothing to do with being a React n00b. That's just me being a blinkered engineer.